dev-tools/packaging/templates/deb/postinstall.sh.tmpl

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
```

Comment:
nit.  should really use `#!/bin/sh` for non-interactive scripts.

Reply:
Thanks! I'll change it.